rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2842,6 +2879,34 @@ def _check_tokens_existence(user_db_cursor: 'DBCursor', expect_removed: bool = F
 
         _check_tokens_existence(user_db_cursor=cursor, expect_removed=True)
 
+        assert cursor.execute(  # events with the correct prefix for coinbase removed
+            'SELECT COUNT(*) FROM history_events WHERE location=? AND event_identifier LIKE ?',
+            (coinbase_location, f'{CB_EVENTS_PREFIX}%'),
+        ).fetchone()[0] == 0
```

Comment:
isn't this incorrect? DIdn't we said to mantain the imported events?

Reply:
there's `CB_EVENT_PREFIX` & `ROTKI_EVENT_PREFIX` -- the one being preserved is `ROTKI_EVENT_PREFIX`